packages/backend/src/index.ts

```diff
@@ -127,6 +128,40 @@ export function initIocContainer(
       replica_addresses: config.tigerbeetleReplicaAddresses
     })
   })
+
+  container.singleton('ratesService', async (deps) => {
+    const config = await deps.use('config')
+    return createRatesService({
+      logger: await deps.use('logger'),
+      exchangeRatesUrl: config.exchangeRatesUrl,
+      exchangeRatesLifetime: config.exchangeRatesLifetime
+    })
+  })
+
+  if (config.enableTelemetry) {
+    container.singleton('fallbackRatesService', async (deps) => {
```

Comment:
I think the name should maybe be `externalRatesService` now that it's a separate thing, and doesn't just live in the telemetry service, thoughts?

Reply:
technically speaking, the rates service that hits the ase provided rates endpoint is also external in rafiki's perspective, right?
That's why i went for fallback

Reply:
and i belive that other occurances of using this external rates service will also be as a fallback to ASE